.github/CODEOWNERS

```diff
@@ -1,13 +1,12 @@
 # Global - fallback to this group if the following groups don't match
-*   @jaynewstrom-stripe @tillh-stripe @ccen-stripe @awush-stripe @jameswoo-stripe @carlosmuvi-stripe
+* @stripe/stripe-android-sdk-reviewers
 
 # Payments SDK
 /payments/ @jaynewstrom-stripe @tillh-stripe @ccen-stripe @jameswoo-stripe
-/payments-model/ @jaynewstrom-stripe @tillh-stripe @ccen-stripe @jameswoo-stripe
```

Comment:
why was this removed?

Reply:
So that it falls back to anyone in the team. Although it's named using the payments domain, `payments-model` acts as a shared module (as it contains models shared across SDKs)